client/src/AddFriend.js

```diff
@@ -25,12 +25,15 @@ class AddFriend extends React.Component {
   componentDidMount () {
     axios.get(`/api${this.props.location.pathname}`)
       .then(res => {
+        console.log(res)
```

Comment:
Is this for testing or should these be deleted?

Reply:
Yup, just for testing purposes. Security-wise, it's not a vulnerability since all the info is in the browser anyway.